cpp/daal/src/algorithms/service_qsort.h

```diff
@@ -1,6 +1,6 @@
 /* file: service_qsort.h */
 /*******************************************************************************
-* Copyright 2014-2020 Intel Corporation
+* Copyright 2020-2020 Intel Corporation
```

Comment:
Just "Copyright 2020 Intel Corporation". See the rest of the library and follow for consistency.